recipes-ni/ni-grpc-device/ni-grpc-device_1.1.bb

```diff
@@ -8,7 +8,13 @@ LIC_FILES_CHKSUM = "\
 	file://${PN}-server/ThirdPartyNotices.txt;md5=6def9ca42f278e76d89243699fae2b67 \
 "
 
+inherit ptest
+
 SRC_URI = "\
+	file://enumerate-device.py \
+	file://run-ptest \
+	file://session_pb2_grpc.py \
+	file://session_pb2.py \
```

Comment:
No these are generated files from python3-grpcio-tools being run with session.proto as input.  I didn't see any reason to force a python compile for this simple test, so I've just included the output from the compile.

Reply:
By the way, the test itself enumerate-devices.py is the same as the sample, but its been "downgraded" to not use F-Strings since python 3.5.5 doesn't support them.  So the test file itself is semi-custom.